garage/envs/mujoco/mujoco_env.py

```diff
@@ -116,12 +116,34 @@ def reset_mujoco(self, init_state=None):
             self.sim.data.ctrl[:] = self.init_ctrl
         else:
             start = 0
-            for datum_name in ["qpos", "qvel", "qacc", "ctrl"]:
-                datum = getattr(self.sim.data, datum_name)
-                datum_dim = datum.shape[0]
-                datum = init_state[start:start + datum_dim]
-                setattr(self.sim.data, datum_name, datum)
-                start += datum_dim
+
+            # update qpos
```

Comment:
* Please use different variables for each case instead of reusing `datum` and `datum_dim`

### Justification
Code should be tolerant to refactoring. If I rearrange some of this code and the variables share names, I might end up with a different result

Reply:
That's an awesome point Ryan! I'll patch that fix right now!